src/react-components/avatar-preview.js

```diff
@@ -107,7 +107,6 @@ export default class AvatarPreview extends Component {
       const dt = clock.getDelta();
       this.mixer && this.mixer.update(dt);
       this.previewRenderer.render(this.scene, this.camera);
-      this.snapshotRenderer.render(this.scene, this.snapshotCamera);
```

Comment:
Woops, this was silly

Reply:
Heh, yeah it was my fault. Just a debug line that I missed.